packages/playground/backend-integration/__tests__/backend-integration.spec.ts

```diff
@@ -32,6 +32,12 @@ if (isBuild) {
     expect(htmlEntry.assets.length).toEqual(1)
   })
 } else {
+  test('No ReferenceError', async () => {
+    browserErrors.forEach((error) => {
+      expect(error.name).not.toBe('ReferenceError')
+    })
+  })
```

Comment:
**suggestion (nitpick):** I would rewrite it to

```suggestion
  test.each(browserErrors, 'No ReferenceError', async (error) => {
    expect(error).not.toThrowError(ReferenceError)
  })
```

Reply:
I dont think `toThrowError` can be used here. Doesnt it need to be like this?
```js
expect(() => { throw error }).not.toThrowError(ReferenceError)
```

Reply:
Yes you are right, I didn't had a VSCode instance while writing the suggestion, sorry ^^